smart-city-electron-dashboard/src/client/src/css/app.css

```diff
@@ -20,7 +20,7 @@
   /* Unique background color for the rest of the application */
   .app-background {
     background-color: #120639; /* Choose your unique background color */
-    min-height: 100vh;
+    min-height: 250vh;
```

Comment:
@PadhmaCebolu Do not change whole application styles.
Please have specific css to you page, this change disturbed whole css of the other pages.